initfiles/sbin/hpcc_setenv.in

```diff
@@ -55,6 +70,10 @@ function pidwait_fn () {
 
     if [[ $TIMEOUT -le 0 ]] && ps -p $WATCH_PID -o command= | grep "${PROCESS}" &>/dev/null ; then
       log "Failed to kill ${PROCESS} with SIGTERM"
+      if [[ $FORCE -eq 1 ]]; then
+        kill -SIGKILL $WATCH_PID > /dev/null 2>&1
```

Comment:
Add a log line here, as in -

log "killing ${PROCESS} (pid: $WATCH_PID) with SIGKILL"